test/controllers/courses_controller_test.rb

```diff
@@ -0,0 +1,58 @@
+require 'test_helper'
+
+class CoursesControllerTest < ActionDispatch::IntegrationTest
+
+    def setup
+        @user = User.create(name: "Courses user", email: "user@courses.ca", password: "coursebar", password_confirmation: "coursebar", role: Role.admin)
+        log_in_user(@user, "coursebar")
+    end
+
+    def teardown
+        log_out_user()
+    end
+
+    test "should get index" do
+        get courses_path
+        assert_response :success
+    end
+
+    test "should redirect to root" do
+        log_out_user()
+
+        get courses_path
+        assert_response :redirect
+
+        log_in_user(@user, "coursebar")
+    end
+
+
+    test "should get review" do
+        get courses_review_path
+        assert_response :success
```

Comment:
should check the content of the response. Eg. draft courses shouldn't show up in response